simple/stats/constants.py

```diff
@@ -22,9 +22,15 @@
 OBSERVATIONS_FILE_NAME = "observations.csv"
 DEBUG_RESOLVE_FILE_NAME = "debug_resolve.csv"
 
-DCID_OVERRIDE_PREFIX = "dcid:"
+# Input columns.
+INPUT_COLUMN_DCID = "dcid"
+INPUT_COLUMN_GEO_ID = "geoId"
```

Comment:
could the geoId be even more generalized here? like we can do "wikidataId" now.

Reply:
Yes, I've made it more generic. PTAL.